registry/purview-registry/registry/purview_registry.py

```diff
@@ -497,15 +497,14 @@ def _register_feathr_feature_types(self):
             force_update=True)
         logger.info("Feathr Feature Type System Initialized.")
 
-    def _upload_entity_batch(self, entity_batch):
+    def _upload_entity_batch(self, entity_batch:list[AtlasEntity]):
+        # we only support entity creation, update is not supported. 
+        # setting lastModifiedTS ==0 will ensure this, if another entity with ts>=1 exist
+        # upload funtion will fail with 412 Precondition fail.
```

Comment:
Since 412 is user error, feathr client should explicitly tell user error details on the operation fails. Show message like The specified feature <> already exists. Probably sql registry already has an error code for this.

Reply:
Ok, will check sql error handling and try raise some right error code. 
Currently on API caller side it is just 500

Reply:
The plan to fix this is merge this first, then Chen will add unified error handling to expose 412 to the customer.